src/main/java/net/datafaker/idnumbers/pt/br/IdNumberGeneratorPtBrUtil.java

```diff
@@ -140,10 +140,9 @@ public static int digit(int verifyingDigit) {
 
     private static String leftPad(char pad, int length, String string) {
         StringBuilder appender = new StringBuilder();
-        for (int i = 0; i < length - string.length(); i++) {
-            appender.append(pad);
-        }
-        return appender.append(string).toString();
+        appender.append(String.valueOf(pad).repeat(Math.max(0, length - string.length())));
+        appender.append(string);
+        return appender.toString();
```

Comment:
no need for `StringBuilder` here

Reply:
Why not? What's the alternative?

Reply:
String.valueOf(pad).repeat(Math.max(0, length - string.length())) + string;

Reply:
Ah, like that. Yes, yes, can do.